deploy/operator.yaml

```diff
@@ -17,13 +17,21 @@ spec:
       containers:
         - name: anthill
           # Replace this with the built image name
-          image: REPLACE_IMAGE
+          image: quay.io/gluster/anthill:latest
           ports:
             - containerPort: 60000
```

Comment:
Any significance for this port ?

Reply:
It's the port for Prometheus metrics, again part of the sdk.